### PR TITLE
wagtail/videoembed: adding missing classname and make sure all page t…

### LINKED ISSUE #554: Video embed: smaller in pages created before update, youtube doesn't work
**URL:** 
pre-update-created: https://ce-dev.liqd.net/ich-bin-eine-simple-page/ 
after-update-created: https://ce-dev.liqd.net/2022_03_test-structured-page/2022-04-blog-page/
**user:** admin
**expected behaviour:** All embed videos look the same
**behaviour:** Videos of same source that were added before all the updates, look smaller and are displayed without a title. Youtube links do not work at all. 
**important screensize:**
**device & browser:** chrome
**Comment/Question:** 
Only Vimeo and Youtube was tested
Screenshot?
**existing page displayed now (vimeo video same as in newly created page):**
image

**newly created page after updates with vimeo and youtube:**
image

### PR DESCRIPTION
**Pull request by khamui**
…ypes are working

closes #554

### CODE REVIEW

File: civic_europe/assets/js/wagtail-videoembed-fix.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+document.addEventListener('DOMContentLoaded', function() {
+  const videoEmbeds = document.querySelectorAll('.rich-text iframe')
+
+  for (const videoEmbed of videoEmbeds) {
+    const parentOfEmbed = videoEmbed.parentElement
+    // following classnames are bs4 classnames
+    parentOfEmbed.classList.add('embed-responsive')
+    parentOfEmbed.classList.add('embed-responsive-16by9')
+  }
+})
```
Comment (phillimorland) [inline civic_europe/assets/js/wagtail-videoembed-fix.js:10]: why do we need this? now all video embeds had rich-tect class name any styling can be done https://github.com/liqd/a4-civic-europe/blob/cc3b6b33b96f79de4ffd7caeb042230d44dff81c/civic_europe/assets/scss/components/_blocks.scss#L245 along with extending any classes we want to add

Comment (khamui) [reply civic_europe/assets/js/wagtail-videoembed-fix.js:10]: maybe it is obsolete, i added to mimic what we have here:

https://github.com/liqd/a4-opin/blob/da3766c759a031649466c74f07e696a8e4b5899d/euth_wagtail/assets/scss/wagtail-blocks/_blocks.scss#L31

**Review by phillimorland — APPROVED** [review]
Nice!

**Comment by phillimorland** [comment]
@khamui looking good but build needs fixing, once it's passing feel free to merge!

**Comment by khamui** [comment]
oh oops, yes fixed now :)

**Pull request merged**